cfeScraping/index.js

```diff
@@ -45,10 +46,49 @@ ScrapPage.prototype.clickButton = async function (id, time) {
     await this.page.click(id);
     console.log('Searching...');
     await this.page.waitForTimeout(time);
-    return console.log('Search succesfull');
+    return console.log('Search successful');
   } catch (err) {
     return console.error('Error: ', err);
   }
 };
 
+// eslint-disable-next-line space-before-function-paren
+ScrapPage.prototype.getDataTable = async function () {
+  try {
+    const data = await this.page.evaluate(
+      (selector = 'table.k-selectable tr') => {
+        const elements = document.querySelectorAll(selector);
+        const inf = [];
+        for (let i = 0; i < elements.length; i++) {
+          const el = elements[i].children;
+          const indArr = [];
+          for (let j = 0; j < elements.length; j++) {
+            indArr.push(el[j].innerText);
+          }
+          inf.push(indArr);
+        }
+        return inf;
+        // eslint-disable-next-line comma-dangle
+      }
+    );
+    data.shift();
```

Comment:
Add comments on your code. to explain why you are gonne shift() this data, & add comments of what this faction should return